cartoframes/layer.py

```diff
@@ -286,15 +285,20 @@ def __init__(self, query, time=None, color=None, size=None,
                 raise ValueError("When time is specified, size can "
                                  "only be a fixed size")
             old_size = size
+            # Default size range, bins, and bin_method
             size = {
                 'range': [5, 25],
-                'bins': 10,
+                'bins': 5,
                 'bin_method': BinMethod.quantiles,
             }
+            # Assign default range and update if min/max given
+            old_size['range'] = size['range']
+            if 'min' in old_size.keys():
+                old_size['range'][0] = old_size['min']
+            if 'max' in old_size.keys():
+                old_size['range'][1] = old_size['max']
+            # Update all the keys in size if they exist in old_size
             size.update(old_size)
-            # Since we're accessing min/max, convert range into a list
-            size['range'] = list(size['range'])
-            self.style_cols[size['column']] = None
```

Comment:
Why was this line removed? It's needed for validation of cartocss later in the `_setup` method.

Reply:
Ah! That's probably a mistake.